File: legacy/quantum-batch/src/main/java/com/quantum/batch/controller/RootController.java
package com.quantum.batch.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

/** 루트 경로 컨트롤러 - Swagger UI로 리다이렉트 */
@Controller
public class RootController {

    /** 루트 경로 접근시 Swagger UI로 리다이렉트 */
    @GetMapping("/")
    public String redirectToSwagger() {
        return "redirect:/swagger-ui.html";
    }

    /** API 루트 경로도 Swagger로 리다이렉트 */
    @GetMapping("/api")
    public String redirectApiToSwagger() {
        return "redirect:/swagger-ui.html";
    }
}

File: quantum-web-api/src/main/kotlin/com/quantum/web/controller/RootController.kt
package com.quantum.web.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.servlet.ModelAndView

/**
 * 루트 경로 컨트롤러
 * 
 * localhost:8080/ 접근 시 Swagger UI로 리다이렉트
 */
@Controller
class RootController {
    
    /**
     * 루트 경로 접근 시 Swagger UI로 리다이렉트
     */
    @GetMapping("/")
    fun redirectToSwagger(): ModelAndView {
        return ModelAndView("redirect:/docs")
    }
}